Dockerfile

```diff
@@ -40,7 +39,7 @@ WORKDIR /code
 COPY requirements.txt .
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-        build-essential curl apt-transport-https libpq-dev netcat jq python3-dev xmlsec1 cron git && \
+        build-essential curl apt-transport-https libpq-dev netcat-traditional jq python3-dev xmlsec1 cron git && \
```

Comment:
MyLA wasn't using `netcat-traditional` until this change was added?  That seems strange to me, because I'm sure it was added a while ago.  (Maybe I'm thinking of another project.)

Reply:
This was added this time since the build was breaking with out it.